### PR TITLE
Use .0 at the end of unplugged levels

### PR DESCRIPTION
**Pull request by etaderhold**
This change makes the level header for unplugged levels have a ".0" suffix (as these are always before level 1 in a lesson). Students who have marked this lesson complete will have that reflected in the table.

image

## Links

[JIRA](https://codedotorg.atlassian.net/browse/TEACH-903)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx [hunk 1]
```diff
@@ -79,6 +81,8 @@ export default function ExpandedProgressColumnHeader({
     [lesson, level, toggleExpandedChoiceLevel, isExpandable]
   );
 
+  console.log(level);
```
Comment (lfryemason) [inline apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx]: Extra log line

File: apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx [hunk 2]
```diff
@@ -66,7 +66,9 @@ export default function ExpandedProgressColumnHeader({
         onClick={() => toggleExpandedChoiceLevel(level)}
       >
         {level.sublevels?.length > 0 && <FontAwesome icon="caret-right" />}
-        <div>{lesson.relative_position + '.' + level.bubbleText}</div>
+        <div>{`${lesson.relative_position}.${
+          level.isUnplugged ? 0 : level.bubbleText
```
Comment (lfryemason) [inline apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx:70]: Since this is returned as undefined, should we instead fix this on the backend?

Comment (etaderhold) [reply apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx:70]: I did some digging and found the `undefined` value is being set on the front end [here](https://github.com/code-dot-org/code-dot-org/blob/a78b1ef6d8cc2a88d10ccb07207791960b0236cd/apps/src/templates/progress/progressHelpers.js#L284). My only concern about changing it there would be unintended side effects on other pages.

Comment (lfryemason) [reply apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx:70]: Seems very intentional then :\

**Review by lfryemason — APPROVED** [review]

**Pull request merged**